controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/user/Roles.java

```diff
@@ -57,10 +54,6 @@ public static Role toRole(TenantName tenant, String roleName) {
     /** Returns the {@link Role} the given tenant and role names correspond to. */
     public static Role toRole(TenantName tenant, ApplicationName application, String roleName) {
         switch (roleName) {
-            case "applicationAdmin": return Role.applicationAdmin(tenant, application);
-            case "applicationOperator": return Role.applicationOperator(tenant, application);
-            case "applicationDeveloper": return Role.applicationDeveloper(tenant, application);
-            case "applicationReader": return Role.applicationReader(tenant, application);
```

Comment:
Will the potential exception be caught by the caller?

Reply:
Yes, see internal PR.